query-request.hh

```diff
@@ -318,7 +320,8 @@ public:
                  utils::UUID query_uuid,
                  query::is_first_page is_first_page,
                  std::optional<query::max_result_size> max_result_size,
-                 uint32_t row_limit_high_bits)
+                 uint32_t row_limit_high_bits,
```

Comment:
(we should have a uint64_t for the row limit, and left the high/low bits for IDL only)

Reply:
This is the IDL constructor. Now it is only used by IDL.
The user-facing constructor does have an `uint64_t` (actually a tagged int).

Reply:
Yes